app/src/main/res/layout/add_meeting_form.xml

```diff
@@ -27,7 +27,8 @@
         android:layout_height="65dp"
         android:layout_marginTop="128dp"
         android:ems="10"
-        android:hint="--/--/--- --:--"
+        android:hint="@string/date"
+        android:autofillHints="date"
```

Comment:
Avoided hardcoded strings and added autofill hints for including an autofilled date